.github/workflows/generate-cve.yml

```diff
@@ -67,17 +67,14 @@ jobs:
 
       - name: Tag
         run: |
-          git config --global user.email ""
-          git config --global user.name "GitHub Actions Bot"
-          git pull
-          git commit --allow-empty -m 'Release ${{ steps.date.outputs.date }}'
-          git push origin main
+          git tag -a cve-${{ steps.date.outputs.date }} -m "release ${{ steps.date.outputs.date }}"
+          git push origin cve-${{ steps.date.outputs.date }}
```

Comment:
Should we be worried about this?
https://github.com/fleetdm/nvd/blob/e0c49a7565bc2410563ca908d91c6294ca0914cb/.github/workflows/generate.yml#L55-L56

Reply:
I've been running this in my personal repo and haven't come across any out of order releases yet.  It's worth noting that releases are in order of tag date, not release date.

https://github.com/mostlikelee/vulnerabilities/releases

Reply:
OK, let's give it a go, we can change/fix if it ever happens.